tools/layer-zero-example/hardhat.config.js

```diff
@@ -0,0 +1,198 @@
+/*-
+ *
+ * Hedera JSON RPC Relay - Hardhat Example
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+require('dotenv').config();
+require("@nomicfoundation/hardhat-toolbox");
+
+module.exports = {
+  solidity: {
+    compilers: [
+      {
+        version: '0.8.22',
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 200,
+          },
+        },
+      },
+    ],
+  },
+  defaultNetwork: 'hedera_testnet',
+  networks: {
+    hedera_testnet: {
+      url: 'https://testnet.hashio.io/api',
+      accounts: [process.env.HEDERA_PK],
+      chainId: 296,
+    },
+    bsc_testnet: {
+      url: "https://data-seed-prebsc-1-s1.binance.org:8545",
+      chainId: 97,
+      gasPrice: 20000000000,
+      accounts: [process.env.BSC_PK]
+    },
+  }
+};
+
+const getEndpointAddress = (network) => {
+  let ENDPOINT_V2;
+
+  if (network === 'hedera_testnet') {
+    ENDPOINT_V2 = '0xbD672D1562Dd32C23B563C989d8140122483631d';
+  } else if (network === 'bsc_testnet') {
+    ENDPOINT_V2 = '0x6EDCE65403992e310A62460808c4b910D972f10f';
+  }
```

Comment:
What are these hard-coded contracts? How to get these? 
Maybe a comment explain these values would be valuable for devs working on or revisiting this in future.

Reply:
They are described here https://docs.layerzero.network/v2/developers/evm/technical-reference/deployed-contracts. Yeah, the PR review requires some LZ knowledge :confused:. I will add the notion link to the description here and write down some comments to the codebase. Thanks!

Reply:
Yes, please add resources and comments for additional context. Otherwise, revisiting this or anyone not familiar LZ knowledge look into it might cause confusion. Thanks much!

Reply:
Added.